index.js

```diff
@@ -63,7 +65,14 @@ class ClientServiceBase {
      * The LedgerClient generated by gRPC library
      * @constant
      */
-    this.ledgerClient = ledgerClient;
+    this.ledgerClient = this.getRequiredProperty_(service, 'ledgerClient');
+
+    /**
+     * The LedgerPrivileged generated by gRPC library
+     * @constant
+     */
+    this.ledgerPrivileged = this.getRequiredProperty_(service,
+        'ledgerPrivileged');
```

Comment:
I guess it is ok to not check if service contains ledgerClient and ledgerPriviledged because ClientServiceBase will not be used by application developers and we (node.js and web SDKs) definitely will pass correctly.